src/beat/mod.rs

```diff
@@ -160,6 +164,13 @@ where
         self
     }
 
+    /// Set the content type of the message body serialization. If set, Beat
+    /// will override the content type set in Signatures.
+    pub fn content_type_override(mut self, content_type: MessageContentType) -> Self {
+        self.config.content_type_override = Some(content_type);
```

Comment:
I think this should just be `content_type` instead of `content_type_override` so we're consistent with `CeleryBuilder`.

```suggestion
    /// Set a default content type of the message body serialization.
    pub fn content_type(mut self, content_type: MessageContentType) -> Self {
        self.config.content_type = Some(content_type);
```